clang/docs/StructureProtection.rst

```diff
@@ -0,0 +1,70 @@
+====================
+Structure Protection
+====================
+
+.. contents::
+   :local:
+
+
+Introduction
+============
+
+Structure protection is an *experimental* mitigation
+against use-after-free vulnerabilities. For
+more information, please see the original `RFC
+<https://discourse.llvm.org/t/rfc-structure-protection-a-family-of-uaf-mitigation-techniques/85555>`_.
+An independent set of documentation will be contributed when the feature
+is promoted to stable.
+
+Usage
+=====
+
+To use structure protection, build your program using one or more of these flags:
+
+- ``-fexperimental-pointer-field-protection``: Enable pointer
```

Comment:
Now that we have another PFP-related flag I decided to rename this one to `-fexperimental-pointer-field-protection-abi` in order to be more precise about exactly what it does.